lib/spack/spack/test/zz/bindist_def.py

```diff
@@ -0,0 +1,416 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
Is there a reason you created the `zz` directory? I think it should be eliminated (and the test files in it placed in `lib/spack/spack/test`)

Also there appears to be duplicate fixture definitions between this and `bindist_rel`

Reply:
I added it under zz to guarantee that it would run last. I was having problems cleaning up after tests plus the tests take minutes because of the builds.

Reply:
We'll want to make sure to clean up after the tests: if you remove the `zz` directory I can have a look at the CI reports to see what they might be complaining about. If you merge from develop it might be that since https://github.com/spack/spack/pull/14309 was merged while this PR was in development it may have eliminated some cross-contamination problems in testing (even if it doesn't though, we'll want to address any remaining issues before merging this).